spec/controllers/presentations_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe PresentationsController, type: :controller do
+  let(:admin_person) { FactoryGirl.build(:person, :admin) }
+
+  before do
+    allow_any_instance_of(PresentationsController).to receive(:pundit_user)
```

Comment:
Yeah because in a controller spec there is no session or concept of a `current_person` so we stub this to return a user that has the priveleges we are trying to test for. The alternative would be a integration test but I feel pretty confident that this hits the spot.